src/lib/dnssd/Resolver.h

```diff
@@ -17,6 +17,7 @@
 
 #pragma once
 
+#include <algorithm>
```

Comment:
I assume this is what causes the big increase in codesize....

Reply:
Updated to not use algorithm. Still used std::swap since that seemed very safe.